{"name": "TC02: Verify team name", "status": "failed", "statusDetails": {"message": "AssertionError: Verify Equals Failed, Actual: test is notequal to Expected: Test api", "trace": "self = <test_cases.test_api.Test_Api object at 0x0000019FF2765AD0>\n\n    @pytest.mark.run_this\n    @allure.title('TC02: Verify team name')\n    @allure.description('This test verify grafana team name')\n    def test_verify_team_member_name(self):\n        nodes = ('teams', 0, 'name')\n        actual = APIFlows.get_value_from_api(nodes)\n>       Verifications.verify_equals(actual, team_name)\n\ntest_api.py:36: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nactual = 'test', expected = 'Test api'\n\n    @staticmethod\n    @allure.step(\"Verify element is equal to expected\")\n    def verify_equals(actual, expected):\n>       assert actual == expected, f'Verify Equals Failed, Actual: {str(actual)} is notequal to Expected: {str(expected)}'\nE       AssertionError: Verify Equals Failed, Actual: test is notequal to Expected: Test api\n\n..\\extensions\\verifications.py:11: AssertionError"}, "description": "This test verify grafana team name", "steps": [{"name": "Get value from Grafana api flow", "status": "passed", "steps": [{"name": "GET Request", "status": "passed", "parameters": [{"name": "path", "value": "'http://localhost:3000/api/teams/search'"}, {"name": "user", "value": "'admin'"}, {"name": "password", "value": "'admin'"}], "start": 1677330847362, "stop": 1677330847397}, {"name": "Extract_value_from_response", "status": "passed", "parameters": [{"name": "response", "value": "<Response [200]>"}, {"name": "nodes", "value": "('teams', 0, 'name')"}], "start": 1677330847397, "stop": 1677330847397}], "parameters": [{"name": "nodes", "value": "('teams', 0, 'name')"}], "start": 1677330847362, "stop": 1677330847397}, {"name": "Verify element is equal to expected", "status": "failed", "statusDetails": {"message": "AssertionError: Verify Equals Failed, Actual: test is notequal to Expected: Test api\n", "trace": "  File \"C:\\Users\\barka\\AppData\\Roaming\\Python\\Python311\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Automation\\Final_Proj_Aut_Py\\extensions\\verifications.py\", line 11, in verify_equals\n    assert actual == expected, f'Verify Equals Failed, Actual: {str(actual)} is notequal to Expected: {str(expected)}'\n"}, "parameters": [{"name": "actual", "value": "'test'"}, {"name": "expected", "value": "'Test api'"}], "start": 1677330847397, "stop": 1677330847397}], "start": 1677330847362, "stop": 1677330847398, "uuid": "f57cf1c2-90f5-41c1-8525-920a9db9e0c0", "historyId": "533424ce3a8d55dcfab24596eb57bef5", "testCaseId": "3eadb1b15bdc21317ca1ddf279911884", "fullName": "test_cases.test_api.Test_Api#test_verify_team_member_name", "labels": [{"name": "tag", "value": "run_this"}, {"name": "parentSuite", "value": "test_cases"}, {"name": "suite", "value": "test_api"}, {"name": "subSuite", "value": "Test_Api"}, {"name": "host", "value": "BarLaptop"}, {"name": "thread", "value": "5320-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_cases.test_api"}]}